js/filter-panel.js

```diff
@@ -49,7 +49,7 @@ FilterPanel.prototype.show = function() {
   accessibility.restoreTabindex(this.$form);
   $('body').addClass('is-showing-filters');
   this.isOpen = true;
-  this.$close.focus();
+  this.$body.find('input').first().focus();
```

Comment:
There are with the transaction period filters. These aren't first, so this was more a thought about future-proofing than addressing a current need.